conmon/conmon.c

```diff
@@ -152,11 +175,11 @@ int main(int argc, char *argv[])
 	/* Create the container */
 	if (terminal) {
 		snprintf(cmd, CMD_SIZE,
-			 "%s create %s --pid-file pidfile --console %s",
-			 runtime_path, cid, slname);
+			 "%s create %s %s --pid-file %s --console %s",
+			 runtime_path, cid, bundle_path, pid_file, slname);
 	} else {
-		snprintf(cmd, CMD_SIZE, "%s create %s --pid-file pidfile",
-			 runtime_path, cid);
+		snprintf(cmd, CMD_SIZE, "%s create %s %s --pid-file %s",
```

Comment:
> What I'm worried about is that we're not handling the error from runc properly.

Out of curiosity, would the error be that we'd be giving runC a second argument, or that it does not parse this second argument as a bundle path ?